docs/source/events/examples/payment-status-change.yaml

```diff
@@ -17,5 +17,17 @@ currency_code: EUR
 # Payment method specific data (stripe intent ID in this case)
 payment_data:
   transaction_id: pi_3NOKIODvbS6ezf4w21zIR6Ft
+# Line items:
+line_items:
+- name: one-off
+  amount: 10
+  quantity: 9
+  tax_rate: 0
+  recurrence_interval: null
+- name: monthly-item
```

Comment:
why is it `monthly-item` with `-item` on the end? One-off is just `one-off`. Do we support annual? If so, say what the possible values could be

Reply:
Ok I think that's fine. In that case, we should say that the value is configured in the UI then and reflects whatever it says there

Reply:
Ok, I’ve added half a sentence to where the property is documented. This is what it looks like now.

![image](https://github.com/moreonion/impact-stack-docs/assets/182122/e92c1065-84c9-40bd-b1e6-b4ccf860284b)